packages/middleware/src/createServer.ts

```diff
@@ -14,20 +14,21 @@ import {
   callApiFunction,
 } from "./handlers";
 
-const app = express();
-app.use(express.json());
-app.use(cookieParser());
-app.use(
-  cors({
-    credentials: true,
-    origin: "http://localhost:3000",
-  })
-);
-app.disable("x-powered-by");
-
 async function createServer<
   TIntegrationContext extends Record<string, IntegrationContext>
 >(config: MiddlewareConfig<TIntegrationContext>): Promise<Express> {
+  const app = express();
+
+  app.use(express.json());
+  app.use(cookieParser());
+  app.use(
+    cors({
+      credentials: true,
+      origin: "http://localhost:3000",
```

Comment:
When I look at it, I feel like we should introduce a second optional argument to the createServer, where we could pass e.g. config to the cors middleware. But it's outside of the scope of this task, just a food for thought